AWMSmonitor.ino

```diff
@@ -0,0 +1,204 @@
+/*
+   Automatic Water Management system(AWMS)
+   Project done at IoTuneZ technologies
+   as part of industrial internship
+   Author: Mr.Greatin Baby and Mr. Abhinand Suresh
+   Code for water level monitoring module
+   LowPower library is used for putting arduino into sleep mode
+   Communication is done using radio wave transmission of packets.
+   LoRa modules are used for transmitting datas.
+*/
+#include <LowPower.h>
+#include <SPI.h>
+#include <LoRa.h>
+#include<String.h>
+#define ECHO_PIN 5            // Echo pin of Ultrasonic sensor
+#define TRIGGER_PIN 6         // Trigger pin of Ultrasonic sensor
+#define LORA_FREQUENCY 433E6  // Set LoRa frequency 
+#define BAUD_RATE 9600        // Serial baud rate
+#define onAck "mon"           // Expected Acknowledgement for motor power on (May be updated after cyphering)
+#define offAck "mof"          // Expected Acknowledgement for motor power off (May be updated after cyphering)

```

Comment:
TODO: Please form a messaging style reply
may be A facy number ( 2bytes): indented client #, 1 byte, status(1 byte), message, message parameters etc. 
For example : AABB, 1A, 01 ( WORKING/ 02, sleeping etc), 05(MOTOR status - 00-normal ACK, 01, configuration, 02 - PAIRING, 03 - TIME, 04, TIME request etc), 01(motor ON)